.github/workflows/prbuild.yaml

```diff
@@ -159,7 +159,7 @@ jobs:
           LOAD_PREBUILT_IMAGE: "true"
           USE_CONTOUR_CONFIGURATION_CRD: ${{ matrix.use_config_crd }}
         run: |
-          export CONTOUR_E2E_IMAGE="ghcr.io/projectcontour/contour:$(ls ./image/contour-*.tar | sed -E 's/.*-(.*).tar/\1/')"
+          export CONTOUR_E2E_IMAGE="ghcr.io/projectcontour/contour:$(ls ./image/contour-*.tar | sed -E 's/contour-(.*).tar/\1/')"
```

Comment:
oops, looks like we're now getting:
```
CONTOUR_E2E_IMAGE=ghcr.io/projectcontour/contour:./image/35907608
```

So need the prefix to be `./image/contour-`

Reply:
did a `.*` again so we didnt have to deal with special characters with the sed